modules/lambdas/main.tf

```diff
@@ -83,6 +83,27 @@ resource "aws_lambda_function" "get_current_archive_list" {
   }
 }
 
+resource "aws_lambda_function" "perform_orca_reconcile" {
```

Comment:
did we decide not to expose internal reconciliation lambdas' arns to end user?

Currently all other lambda arns are added to lambdas/output.tf, output.tf , modules/orca.tf

Reply:
We only expose 8 lambda arns in outputs.tf. Ive added to lambdas/output.tf since we'll need it for workflows.